src/settings/Models/common/Metadata.test.js

```diff
@@ -0,0 +1,16 @@
+import Metadata from './Metadata';
+import { commonClassCheck } from '../../../../test/jest/helpers/utils';
+
+const testMetadata = commonClassCheck.bind(null, Metadata);
+
+describe('Metadata', () => {
+  testMetadata();
+
+  testMetadata({ createdByUserId: 'testCreatedByUserId' });
+
+  testMetadata({ createdDate: 'testCreatedDate' });
+
+  testMetadata({ updatedByUserId: 'testUpdatedByUserId' });
+
+  testMetadata({ updatedDate: 'testUpdatedDate' });
```

Comment:
Is it possible to combine it to the following view:

```suggestion
  testMetadata({
      createdByUserId: 'testCreatedByUserId',
      createdDate: 'testCreatedDate' ,
      updatedByUserId: 'testUpdatedByUserId',
      updatedDate: 'testUpdatedDate'
  });
```

Reply:
Done